src/modules/todosclass.js

```diff
@@ -0,0 +1,57 @@
+import renderTodoList from './rendertodolist.js';
+
+const input = document.querySelector('#new-todo');
+class Todos {
+  // array to hold todos list
+  todosArr = [];
+
+  // check if todos present in localstorage
+  check() {
+    if (localStorage.getItem('todoStore')) {
+      this.todosArr = JSON.parse(localStorage.getItem('todoStore'));
+    }
+  }
+
+  // Add new todo
+  addNewTodo() {
+    const newTodos = {
+      id: Date.now(),
+      description: input.value,
+      isCheck: false,
+    };
```

Comment:
- [ ] Nice approach with the `id` 👏. However, you still need to add an `index` and set it to the length of the todosArr at the time the new todo item was created. For instance, if a 5th task were added to the list, the index of that task would be equal to 5.

**Please note that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.** 🎯